Model01-Firmware.ino

```diff
@@ -332,7 +358,16 @@ KALEIDOSCOPE_INIT_PLUGINS(
 
   // The HostPowerManagement plugin allows us to turn LEDs off when then host
   // goes to sleep, and resume them when it wakes up.
-  HostPowerManagement
+  HostPowerManagement,
+
+  // The MagicCombo plugin lets you use key combinations to trigger custom
+  // actions - a bit like Macros, but use multiple keys instead of one.
+  MagicCombo,
+
+  // The USBQuriks plugin lets you do some quirky business with USB, such as
```

Comment:
Quirks rather than Quriks.

I'd probably drop the 'quirky business' joke in favor of a tighter description.